### DISCUSSION TITLE
How to structure a dbt project in an existing Dagster project

### DISCUSSION THREAD

**Discussion by rexledesma** [dagster-io/dagster#12248] [Q&A] [answered]
Imagine you already have an existing directory structure that includes your Dagster definitions. You then want to include a dbt project in order orchestrate the materialization of models or other operations related to your dbt project.

How do you include this dbt project in your existing directory structure?

**Comment by rexledesma** [comment, marked as answer]
In our example [dbt + dagster](https://docs.dagster.io/integrations/dbt) project, the dbt project is kept in the top level of your Dagster project.

There, we have two folders ([code](https://github.com/dagster-io/dagster/tree/master/examples/tutorial_dbt_dagster)) to illustrate this:
- `jaffle_shop`: the dbt project
- `jaffle_dagster`: the Dagster project that loads the dbt project as Dagster definitions.

**Comment by slopp** [comment]
See also: https://github.com/dagster-io/dagster/discussions/13767 for different deployment options if your dbt project lives outside of the dagster project.
